From 5465a99ff0c6d7e3124841f5a31471ab9003617b Mon Sep 17 00:00:00 2001
From: ksrt12 <kazakov12stepan2012@live.ru>
Date: Wed, 16 May 2016 20:25:12 +0700
Subject: [PATCH] bay dirty

---
 scripts/setlocalversion | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 63d91e2..a7fe39e 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -72,11 +72,6 @@ scm_version()
 			printf -- '-svn%s' "`git svn find-rev $head`"
 		fi
 
-		# Check for uncommitted changes
-		if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
-			printf '%s' -dirty
-		fi
-
 		# All done with git
 		return
 	fi
@@ -95,12 +90,6 @@ scm_version()
 			fi
 		fi
 
-		# Are there uncommitted changes?
-		# These are represented by + after the changeset id.
-		case "$hgid" in
-			*+|*+\ *) printf '%s' -dirty ;;
-		esac
-
 		# All done with mercurial
 		return
 	fi
-- 
2.8.2

