From ecb5a36c02f7e090b92c56d3b9cadfe10f7c2815 Mon Sep 17 00:00:00 2001
From: ksrt12 <kazakov12stepan2012@live.ru>
Date: Thu, 21 Apr 2016 20:26:54 +0700
Subject: [PATCH] ksrt12 patch

---
 scripts/mkcompile_h     |  3 ++-
 scripts/setlocalversion | 11 -----------
 2 files changed, 2 insertions(+), 12 deletions(-)

diff --git a/scripts/mkcompile_h b/scripts/mkcompile_h
index 6fdc97e..afd6d31 100755
--- a/scripts/mkcompile_h
+++ b/scripts/mkcompile_h
@@ -38,7 +38,8 @@ else
 fi
 
 if [ -z "$KBUILD_BUILD_TIMESTAMP" ]; then
-	TIMESTAMP=`date`
+	TIMESTAMP=$(echo `date` | sed -e 's/+07/ALEYSK/g')
+	vecho "  TMSTMP  $TIMESTAMP"
 else
 	TIMESTAMP=$KBUILD_BUILD_TIMESTAMP
 fi
diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 63d91e2..a7fe39e 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -72,11 +72,6 @@ scm_version()
 			printf -- '-svn%s' "`git svn find-rev $head`"
 		fi
 
-		# Check for uncommitted changes
-		if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
-			printf '%s' -dirty
-		fi
-
 		# All done with git
 		return
 	fi
@@ -95,12 +90,6 @@ scm_version()
 			fi
 		fi
 
-		# Are there uncommitted changes?
-		# These are represented by + after the changeset id.
-		case "$hgid" in
-			*+|*+\ *) printf '%s' -dirty ;;
-		esac
-
 		# All done with mercurial
 		return
 	fi
-- 
2.8.1

